fairscale/nn/misc/flatten_params_wrapper.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) Tongzhou Wang
+# Licensed under the MIT License.
```

Comment:
You'll need to update the NOTICE file and the LICENSE and REFERENCES section of README.md

Reply:
Done!